src/containers/Network/UpgradeStatus.tsx

```diff
@@ -154,11 +155,15 @@ export const UpgradeStatus = () => {
           newValidatorList[validator.signing_key] = validator
         })
 
-        setVList(newValidatorList)
-        setUnlCount(
-          validators.filter((validator) => Boolean(validator.unl)).length,
-        )
-        return Object.values(newValidatorList)
+        if (validators != null) {
```

Comment:
Shouldn't this always be update even if `validators` is null?

Reply:
`validators` can only be `null` if the VHS link is broken, otherwise it would be an array (can be empty). Not sure what you meant by handles updates